draft-ietf-quic-recovery.md

```diff
@@ -392,6 +357,72 @@ Pseudocode for OnPacketAcked follows:
      sent_packets.remove(acked_packet);
 ~~~
 
+## Setting the Loss Detection Alarm
+
+QUIC loss detection uses a single alarm for all timer-based loss detection.  The
+duration of the alarm is based on the alarm's mode, which is set in the packet
+and timer events further below.  The function SetLossDetectionAlarm defined
+below shows how the single timer is set based on the alarm mode.
+
+### Handshake Packets
+
+The initial client hello has no prior RTT sample.  A client SHOULD remember
+the previous RTT it observed when resumption is attempted and use that for an
+initial RTT value.  If no previous RTT is available, the initial RTT defaults
+to 200ms.  Once an RTT measurement is taken, it MUST replace initial_rtt.
+
+The client hello is retransmitted via a timer, similar to a tail loss probe, but
```

Comment:
An ACK for the ClientHello isn't needed, but it does contain some useful information (if you don't care about provenance, that is).

@martinduke, your use of "timeout" in that proposed text is problematic I think.  It could refer to the handshake timing out, but I think that you simply intended to retransmit on a doubling interval (i.e., 200ms, 400ms, 600ms, etc...).

I wonder if some amount of randomization on that retransmission is necessary to avoid synchronizing retransmissions across a large cohort of clients that are simultaneously disconnected, or whether variations in RTT will be enough to deal with that.